### PR TITLE
Componets

### PR DESCRIPTION
**Pull request by kkmanuu**
Hello there;

- All the boilerplate code has been removed, and a new directory "components" has been created. 
- Inside, I  have implemented the "Calculator.js" React component, adhering to the provided design, including JSX and CSS for the UI.  
- I  have utilized functional components and have included at least one child component for better modularity.

### CODE REVIEW

**Review by iambenkis — COMMENTED** [review]
Hi @kkmanuu  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 

- [ ] You have submitted a project with a README that does not have all the relevant headers 👍 , KIndly refer to this template and make sure you have them 👍 


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@iambenkis` in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/Components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        <button type="button" className="operation" onClick={() => handleClick('+')}>+</button>
+      </div>
+      <div className="buttons-div">
+        <button type="button" className="zero" onClick={() => handleClick('0')}>0</button>
+        <button type="button" onClick={() => handleClick('.')}>.</button>
+        <button type="button" className="operation" onClick={() => handleClick('=')}>=</button>
+      </div>
+    </div>
+  );
+};
```
Comment (roshan-bajgain) [inline src/Components/Calculator.js]: - [ ] According to the project requirement your Calculator.js should have at least 1 child component. So I would suggest you please make one child component. Example: You can make different component for buttons and import and use in calculator.js

Comment (roshan-bajgain) [inline src/Components/Calculator.js]: - [ ] The `Calculator.js` component should match the following given design. please see the design as implemented it
### Your design
his

### Original design
original

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @kkmanuu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Good job on implementing the requirement ✔️ 
## Required Changes ♻️
- [ ] I cannot leave comment there so I am leaving it here please implement it all the boilerplate that got created by the CRA (*texts*, *images*, and *styles*) should be deleted. Please delete from public folder. Also delete the test file and unwanted file in `src `folder.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      <button type="button">7</button>
+      <button type="button">8</button>
+      <button type="button">9</button>
+      <button type="button" className="operation">x</button>
+    </div>
+    <div className="buttons-div">
+      <button type="button">4</button>
+      <button type="button">5</button>
+      <button type="button">6</button>
+      <button type="button" className="operation">-</button>
```
Comment (ghost) [inline src/Components/Calculator.js:24]: - [ ] Good work up to this point. However, it looks like your design was affected after working on the changes requested. Kindly check it out

image

**Review by ghost — CHANGES_REQUESTED** [review]
Hi @kkmanuu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Gitflow
✅ No linter error


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Calculator.js (hunk 2 above)
Comment (kkmanuu) [reply src/Components/Calculator.js:24]: @frank1738 I have implemented the required changes when you inspect the design looks awesome as the required changes you mention. I can't wait to be approved

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @kkmanuu, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title and summary. ✔️
-  project should have a directory called `components` ✔️
- `Calculator.js` file inside the components directory. ✔️
- `Calculator.js` component should be a **functional** component. ✔️
- `Calculator.js` component match UI design ✔️
- Good use of Git Flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**